forc/src/utils/defaults.rs

```diff
@@ -49,24 +49,62 @@ path = "tests/harness.rs"
 }
 
 pub(crate) fn default_program() -> String {
-    r#"script;
+    r#"contract;
 
-fn main() {
+abi MyContract {
+    fn test_function() -> bool;
+}
 
+impl MyContract for Contract {
+    fn test_function() -> bool {
+        true
+    }
 }
 "#
     .into()
 }
 
-pub(crate) fn default_test_program() -> String {
-    r#"
+// TODO Ideally after (instance, id) it should link to the The Fuels-rs Book
+// to provide further information for writing tests/working with sway
+pub(crate) fn default_test_program(project_name: &str) -> String {
+    format!(
+        "{}{}{}{}{}",
+        r#"use fuel_tx::{ContractId, Salt};
+use fuels_abigen_macro::abigen;
+use fuels_contract::{contract::Contract, parameters::TxParameters};
+use fuels_signers::util::test_helpers;
+
+// Load abi from json
+abigen!(MyContract, "out/debug/"#,
```

Comment:
Rather than using the literals `out/debug/`, `-abi.json`, ./out/debug/`, and `.bin`, can you use some constants here? I imagine that these are constants wherever they're used to generate the files, so use those constants.

Reply:
Actually, looking through the code I see they're actually literals elsewhere. Can you file an issue to make them into constants everywhere?